server/tests/integration/graphql.test.js

```diff
@@ -133,10 +133,13 @@ describe('graphql', () => {
         const excludedTypeNames = [
```

Comment:
I excluded the new types from the GraphQL test because I'd like to understand how the test works.

cc: @alflennik